clients/privacy-center/components/modals/consent-request-modal/ConsentRequestModal.tsx

```diff
@@ -21,9 +21,8 @@ export const useConsentRequestModal = () => {
   );
 
   const successHandler = useCallback(() => {
-    setConsentRequestId(consentRequestId);
     router.push("consent");
```

Comment:
a toast here might be nice. I think they do persist across route changes—if you trigger a toast on this page https://chakra-ui.com/docs/components/toast and then click on another link, the toast will still stick around)

Reply:
ohh nice! I specifically didn't use a toast because of the page change. I didn't know they persisted across pages

Reply:
I'm running into some weird issues getting the toast working. Are you okay with it being addressed in a follow up?

Reply:
The issue is react no-op errors. I'm not quite sure what's causing the no-op

Reply:
yeah no problem! let's do a follow up issue since iirc there are a few places that might be nice to have a toast